src/chainparamsbase.cpp

```diff
@@ -91,6 +108,7 @@ void SelectBaseParams(CBaseChainParams::Network network)
 
 CBaseChainParams::Network NetworkIdFromCommandLine()
 {
+    bool fBlockRegTest = GetBoolArg("-blockregtest", false);
```

Comment:
Ok. There's no use for this, I only added it because the others are there. I'll kill it.